BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/polar_plotter_v2.py

```diff
@@ -35,12 +35,17 @@
 parser.add_argument("freq_num", help="Frequency number (1-60) to plot", type=int)
 parser.add_argument("NPOP", help="Number of individuals in a generation (ex: 10)", type=int)
 parser.add_argument("gen", help="Generation number (ex: 0)", type=int)
+parser.add_argument("SYMMETRY", help="Symmetry of antenna (ex: 2)", type=int)
```

Comment:
Any reason why this is ALLCAPS?

Reply:
The variable, along with most of the settings variables, in the main bash script is named in all caps, so I kept the same convention when passing it into the Python script to show it's an outside setting. Do you think it would be better to rename it in the python script or keep the same name across the scripts it's used in?

Reply:
It feels a bit weird that different (arguably internal) variables use different conventions.  If the convention is well established enough that changing the case might create confusion, then it makes more sense.

One thing you can do is adding a `metavar`:
```suggestion
parser.add_argument("symmetry", help="Symmetry of antenna (ex: 2)", type=int, metavar='SYMMETRY')
```
and in the output it will look like:
```
usage: [-h] SYMMETRY

positional arguments:
  SYMMETRY    Symmetry of antenna (ex: 2)
```
while keeping the internal variable as `args.symmetry`.

This will make it clearer for users of the script while providing consistency to the developers.

Reply:
Alright, I just made a commit to change the argument conventions to match the python ones. Thanks for the suggestions!